test/integration/application.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Application', function() {
+  let app;
+  let fooView;
+  let barView;
+
+  beforeEach(function() {
+    cy.server();
+
+    cy.visitMn(function({_, Backbone, Marionette}) {
+      const Application = Marionette.Application.extend({
+        region: '#app-hook',
+      });
+
+      fooView = new Marionette.View({
+        template: _.template('Application region content')
+      });
+      barView = new Marionette.View({
+        template: _.template('<h1 class="title">Application region content</h1>')
+      });
+
+      app = new Application();
+      app.start();
+    });
+  });
+
+  it('should show a view without additional arguments', function() {
+    app.showView(fooView);
+    cy.get('#app-hook').should('contain', 'Application region content');
+  });
+
+  it('should show a view with additional arguments', function() {
+    app.showView(barView, {
+      selector: '.title',
```

Comment:
What is `selector`?  just filler?  It seems wrong.. if it's region related `selector` was deprecated to `el`, but in this case I don't think it is doing anything?